e2e/tests/admin_api/get_user_by_loginid_test.go

```diff
@@ -0,0 +1,292 @@
+	Convey("GetUserByLoginIDByUsername", t, func() {
+		cmd, err := testrunner.NewEnd2EndCmd(testrunner.NewEnd2EndCmdOptions{
+			TestCase: &testrunner.TestCase{
+				Path: "admin_api/get_user_by_loginid_test.go",
+			},
+			Test: t,
+		})
+		So(err, ShouldBeNil)
+
+		// Prepare testing user
+		err = cmd.ExecuteSQLInsertUpdateFile("../../tests/admin_api/get_users_admin_api_user.sql")
+		So(err, ShouldBeNil)
+
+		rawResp, err := cmd.Client.GraphQLAPIRaw(nil, nil, cmd.AppID, e2eclient.GraphQLAPIRequest{
+			Query: `
+				query {
+					getUsersByLoginID(loginIDKey:"username", loginIDValue: "e2e_admin_api_get_users_username") {
+							createdAt
+							standardAttributes
+							lastLoginAt
+					}
+				}
+	
+			`,
+		})
+		So(err, ShouldBeNil)
+		So(rawResp, ShouldEqualJSON, `
+			{
+				"data": {
+					"getUsersByLoginID": {
+						"createdAt": "2024-08-27T07:51:42.040683Z",
+						"lastLoginAt": null,
+						"standardAttributes": {
+							"preferred_username": "e2e_admin_api_get_users_username",
+							"updated_at": 1724745102
+						}
+					}
+				}
+			}
+		`)
+	})
+}
+
+func TestGetUserByLoginIDNotFound(t *testing.T) {
+	Convey("GetUserByLoginIDNotFound", t, func() {
+		cmd, err := testrunner.NewEnd2EndCmd(testrunner.NewEnd2EndCmdOptions{
+			TestCase: &testrunner.TestCase{
+				Path: "admin_api/get_user_by_loginid_test.go",
+			},
+			Test: t,
+		})
+		So(err, ShouldBeNil)
+
+		// Prepare testing user
+		err = cmd.ExecuteSQLInsertUpdateFile("../../tests/admin_api/get_users_admin_api_user.sql")
+		So(err, ShouldBeNil)
+
+		rawResp, err := cmd.Client.GraphQLAPIRaw(nil, nil, cmd.AppID, e2eclient.GraphQLAPIRequest{
+			Query: `
+				query {
+					getUsersByLoginID(loginIDKey:"email", loginIDValue: "e2e_admin_api_get_users_not_found@example.com") {
+							createdAt
+							standardAttributes
+							lastLoginAt
+					}
+				}
+	
+			`,
+		})
+		So(err, ShouldBeNil)
+		So(rawResp, ShouldEqualJSON, `
+			{
+				"data": {
+					"getUsersByLoginID": null
+				}
+			}
+		`)
+	})
+}
+
+func TestGetUserByLoginIDInvalidLoginIDValue(t *testing.T) {
+	Convey("GetUserByLoginIDInvalidLoginIDValue", t, func() {
+		cmd, err := testrunner.NewEnd2EndCmd(testrunner.NewEnd2EndCmdOptions{
+			TestCase: &testrunner.TestCase{
+				Path: "admin_api/get_user_by_loginid_test.go",
+			},
+			Test: t,
+		})
+		So(err, ShouldBeNil)
+
+		// Prepare testing user
+		err = cmd.ExecuteSQLInsertUpdateFile("../../tests/admin_api/get_users_admin_api_user.sql")
+		So(err, ShouldBeNil)
+
+		rawResp, err := cmd.Client.GraphQLAPIRaw(nil, nil, cmd.AppID, e2eclient.GraphQLAPIRequest{
+			Query: `
+				query {
+					getUsersByLoginID(loginIDKey:"email", loginIDValue: "bad_email_address") {
+							id
+							createdAt
+							standardAttributes
+							lastLoginAt
+					}
+				}
+	
+			`,
+		})
+		So(err, ShouldBeNil)
+		So(rawResp, ShouldEqualJSON, `
+			{
+				"data": {
+					"getUsersByLoginID": null
+				},
+				"errors": [
+					{
+						"message": "invalid Login ID key",
```

Comment:
The message is wrong. It should be `invalid Login ID value`

Reply:
fixed e2e test